setup.py

```diff
@@ -48,14 +48,12 @@
           'License :: OSI Approved :: BSD License',
           'Operating System :: OS Independent',
           'Programming Language :: Python',
-          'Programming Language :: Python :: 2',
-          'Programming Language :: Python :: 2.7',
           'Programming Language :: Python :: 3',
           'Topic :: Software Development :: Libraries :: Python Modules',
           'Topic :: Scientific/Engineering :: Astronomy'],
       platforms=['OS Independent'],
       keywords='meerkat ska',
-      python_requires='>=2.7,!=3.0,!=3.1,!=3.2',
+      python_requires='>=3.5',
```

Comment:
This feels like a separate PR... Or maybe not, given the age of Python 2 :-D

Reply:
The alternative is to add a bunch of code to test whether the installed version of katsdptelstate supports indexed keys so that we can continue to support the old Python 2 version of katsdptelstate.